test/models/content_view_version_test.rb

```diff
@@ -67,6 +68,32 @@ def test_docker_count
       assert_equal image_count, cvv.docker_image_count
       assert_equal tag_count, cvv.docker_tag_count
     end
+
+    def test_search_equal_version
+      assert_includes ContentViewVersion.search_for("version = 1.0"), @cvv
+      assert_includes ContentViewVersion.search_for("version = 1"), @cvv
+    end
+
+    def test_search_compare_version
+      assert_includes ContentViewVersion.search_for("version >= 1.0"), @cvv
+      assert_includes ContentViewVersion.search_for("version <= 1.0"), @cvv
+      assert_includes ContentViewVersion.search_for("version < 1.1"), @cvv
+      assert_includes ContentViewVersion.search_for("version < 2"), @cvv
```

Comment:
What do you think about an assert that ensures the `>` and/or `<` operators return multiple results?